sklearn/utils/tests/test_testing.py

```diff
@@ -602,20 +602,21 @@ def test_check_docstring_parameters(mock_meta):
             "In function: "
             + f"sklearn.utils.tests.test_testing.{mock_meta_name}."
             + "predict_proba",
-            "Parameters in function docstring have less items w.r.t. function"
-            " signature, first missing item: X",
-            "Full diff:",
-            "- ['X']",
-            "+ []",
+            "potentially wrong underline length... ",
+            "Parameters ",
+            "--------- in ",
+            "This is available only if delegate has predict_proba.",
```

Comment:
This it the new error message, when the `Parameters` header does not have the correct underline length.